tasks/rulegen/src/main.rs

```diff
@@ -175,6 +176,17 @@ fn format_code_snippet(code: &str) -> String {
     format!("r#\"{}\"#", code.replace("\\\"", "\""))
 }
 
+// TODO: handle `noFormat`(in typescript-eslint)
+fn format_tagged_template_expression(tag_expr: &TaggedTemplateExpression) -> Option<String> {
+    if tag_expr.tag.is_specific_member_access("String", "raw") {
+        tag_expr.quasi.quasis.first().map(|quasi| format!("r#\"{}\"#", quasi.value.raw))
+    } else if tag_expr.tag.is_specific_id("dedent") || tag_expr.tag.is_specific_id("outdent") {
+        tag_expr.quasi.quasis.first().map(|quasi| util::dedent(&quasi.value.raw).to_string())
```

Comment:
should we throw an err if there is > 1 quasis

Reply:
i'm not sure, I haven't seen any cases so far where it made a big difference, if it's highly dynamic code we probably have to review all of the tests manually anyways. let's keep it as-is and see how it goes